docs/wiki/protocol/overview.md

```diff
@@ -0,0 +1,74 @@
+# Ethereum Overview
+
+## Network coming online and peer-to-peer computing
+
+The concept of peer-to-peer (P2P) networks, where computers communicate and share resources directly without a central authority, has a rich history dating back to the 1960s. Pioneering projects like the ARPANET project, switched on in late October 1969 became the first wide-area packet-switched network, laid the groundwork for this decentralized approach. ARPANET demonstrated the model of distributed communication, where "every participating node could request and serve content".
+
+![ARPANET-map](img/overview/ARPANET-map.jpg)
+> The map of ARPANET in the 1960s
+
+This decentralized philosophy gained significant traction in the late 1990s with the rise of file-sharing platforms like [Napster](https://en.wikipedia.org/wiki/Napster) (1999). These platforms challenged the traditional client-server model, where users relied on a central server to access data. Napster, and subsequent platforms like [BitTorrent](https://en.wikipedia.org/wiki/BitTorrent) (2001), sparked new conversations and philosophies highlighting the potential for a more democratic and distributed online interaction.
+
+## Introduction of Bitcoin
+
+Introduced in a 2008 paper by Satoshi Nakamoto titled ["Bitcoin: A Peer-to-Peer Electronic Cash System"](https://bitcoin.org/bitcoin.pdf), blockchain established a distributed ledger system where data is cryptographically linked in chronological blocks. This innovation empowered the creation of Bitcoin, the first decentralized digital currency, eliminating the need for trusted third-party intermediaries like banks.
+
+Bitcoin's focus on secure digital payments, while revolutionary, limited its ability to run more complex applications. Similar to how modern applications rely on HTTP for communication, early projects attempted to expand Bitcoin's functionality by leveraging its secure cryptographic and peer-to-peer foundation.
```

Comment:
I don't really get the HTTP comparison here 😅

Bitcoin transactions are based on satisfying a simple script which doesn't allow full programmability, it is limited only to enable transfers. Projects inspired by this technology try to utilize it for different use cases..

Reply:
I was going for an analogy between how modern apps ( mobile, web) use HTTP as a buliding block, namecoin etc tried to use bitcoin as a building block. Does that make sense?

Reply:
Ah I see what you mean. I would rephrase it, something along what I said above, 'other projects utilize the fundamental technology of btc to apply to other use cases, similarly how different applications use the internet protocol...'

Reply:
The important part here is that bitcoin scripting is limited and other uses of its blockchain proved impractical requiring drastic changes which are not compatible with the base protocol, therefore new separate blockchains like Eth spin out to it's own projects. Hope it makes sense :)